### ISSUE TITLE
Tables cannot be set to 100% width

### ISSUE THREAD

**Issue by Tset-Noitamotua** [squidfunk/mkdocs-material#175]
How can I make the generated tables have the same width as other boxes - e.g. use all the space until the green line in screenshot below

![image](https://cloud.githubusercontent.com/assets/5635462/23171439/c9456fee-f852-11e6-98fa-00006cce5a95.png)

Cheers
Tset

**Comment by squidfunk** [comment]
Tables are a bit hacky in Material, due to the way overflow is handled on small screens. I will investigate, but it may take some time. Just setting the table to 100% doesn't do the trick, obviously.

**Comment by Tset-Noitamotua** [comment]
I hoped that adding custom stylesheet through `extra_css` would  do the trick :)

**Comment by squidfunk** [comment]
I refactored table styling in #185. Setting tables to full-width is now as easy as:

``` css
.md-typeset__table {
   width: 100%;
}
```

However, doing this you lose one thing: the right padding on overflowing tables on mobile - the red area will not be there anymore, thus resulting in the table having no right padding **when overflowing**:

<img width="400" alt="bildschirmfoto 2017-02-25 um 11 31 56" src="https://cloud.githubusercontent.com/assets/932156/23330449/2e144ff0-fb4e-11e6-809d-1a8a0177576e.png">

The default way how tables behave is unaltered, so there are no changes for users that don't override `md-typeset__table` with their own CSS.

**Comment by squidfunk** [comment]
Closed the issue again, duh. I reopen it and leave it open until it is released.

**Comment by Tset-Noitamotua** [comment]
Great! I'll try it out then back in office on monday.

**Comment by squidfunk** [comment]
Fix released in 1.1.0

**Comment by Tset-Noitamotua** [comment]
It's really as easy as adding following lines to `extra.css` refered in `mkdocs.yml`:

```css
.md-typeset__table {
   min-width: 100%;
}
```

In Chrome I had to clear the browser cache to see the change take effect.

I like it! Thanks a lot @squidfunk 

![image](https://cloud.githubusercontent.com/assets/5635462/23361372/aaf27b52-fcf0-11e6-8bed-d4712d32e20f.png)

**Comment by squidfunk** [comment]
> In Chrome I had to clear the browser cache to see the change take effect.

This is almost certainly not related to Material but to your `extra_css` which has no revision appended for cache busting.

**Comment by biochimia** [comment]
Is there a reason why the suggested fix can't be applied in the base theme only for non-mobile devices?

**Comment by rjpereira** [comment]
Hi. This fix used to work with 4.6.3 but now that I upgraded to 5.1.1. seems broken again. Any recommendation on how to get the 100% width tables with 5.1.1 ? Thanks

**Comment by martinbira** [comment]
> Hi. This fix used to work with 4.6.3 but now that I upgraded to 5.1.1. seems broken again. Any recommendation on how to get the 100% width tables with 5.1.1 ? Thanks

In your `extra_css` you can override the default display setting as follows:
```
.md-typeset table:not([class]) {
    display: table;
}
```

**Comment by rjpereira** [comment]
Thanks. To be clear for others, it worked properly when both are in extra_css:

```
.md-typeset__table {
   min-width: 100%;
}

.md-typeset table:not([class]) {
    display: table;
}

```
